src/components/CreateGroupForm/CreateGroupForm.tsx

```diff
@@ -135,11 +131,11 @@ export default function CreateGroupForm() {
     }
 
     if (
-      inputValues.L === 'off' &&
-      inputValues.M === 'off' &&
-      inputValues.X === 'off' &&
-      inputValues.J === 'off' &&
-      inputValues.V === 'off'
+      inputValues.LU === 'off' &&
+      inputValues.MA === 'off' &&
+      inputValues.MI === 'off' &&
+      inputValues.JU === 'off' &&
+      inputValues.VI === 'off'
```

Comment:
Just for the future. What are your thoughts about having an array that stores ['LU', 'MA'..., 'VI] and so you can do a daysArray.every((day) => inputValues[day] === 'off')